tetris.js

```diff
@@ -203,6 +175,34 @@ module.exports = {
 		}
 
 	},
+
+	checkThatTheFieldIsFree: function (arrayOfCoordinates, field) {
+		if (!this.allItemsOnTheField(arrayOfCoordinates, field)) {
+			return false
+		}
+		for (let coordinate of arrayOfCoordinates) {
+			if (field[coordinate.y][coordinate.x]) {
+				return false
+			}
+		}
+		return true;
+	},
+
+	allItemsOnTheField: function (arrayOfCoordinates, field) {
+		let lengthField_X = field[0].length;
+		let lengthField_Y = field.length;
+		for (let coordinate of arrayOfCoordinates) {
+			if (
```

Comment:
join this line and the next

Reply:
I don't understend  :(

Reply:
Try
https://stackoverflow.com/a/51762551